worlds/ff6wc/__init__.py

```diff
@@ -164,6 +166,10 @@ def create_location(self, name: str, id: Union[int, None], parent: Region) -> FF
         return return_location
 
     def generate_early(self):
+        # if requested to exclude the Zozo Clock Chest, add to exclude_locations
+        if self.options.ZozoClockChestExclude:
+            self.options.exclude_locations.value.add("Zozo Clock Puzzle")
```

Comment:
I think we also have to force the Zozo Clock Puzzle chest to be a local item if we're going to take suggestions from Discord. I don't really know how to do that here though...I know that there's a vanilla Morph option for Super Metroid, but I only see this in the _init_.py for sm:
        if (self.variaRando.args.morphPlacement == "early"):
            self.multiworld.local_early_items[self.player]['Morph Ball'] = 1
I think Morph Ball is the location name, so perhaps I can copy the 2nd line to make the location as a local early item, but I'm unclear as to how the rest of the AP processing works, so some advice here would be great.

Reply:
I'm not sure whether it's a good idea to put workarounds here for problems in other games.

There might be a solution with the more advanced options in yamls for players who are worried about it.
(A player could plando an item into their zozo clock location to make sure that someone else's item isn't there.)